libbpf-tools/opensnoop.bpf.c

```diff
@@ -39,8 +44,8 @@ int trace_filtered(u32 tgid, u32 pid)
 		return 1;
 	if (targ_pid && targ_pid != pid)
 		return 1;
-	if (targ_uid) {
-		uid = bpf_get_current_uid_gid();
+	if (valid_uid(targ_uid)) {
+		uid = bpf_get_current_uid_gid() & 0xffffffff;
```

Comment:
nit: `(u32)bpf_get_current_uid_gid()` is shorter and a bit nicer (and harder to get wrong w.r.t. to number of 'f's :) )